src/System.Management.Automation/engine/Subsystem/FeedbackSubsystem/IFeedbackProvider.cs

```diff
@@ -291,6 +292,7 @@ internal GeneralCommandErrorFeedback()
 
                 if (results.Count > 0)
                 {
+                    ApplicationInsightsTelemetry.SendUseTelemetry(this.GetType().Name, "ResultsSuggested", Convert.ToDouble(results.Count));
```

Comment:
It feels to me the number of results is not very useful data for us to collect. Maybe we just collect whether this feature gets used by sending the telemetry here instead? https://github.com/PowerShell/PowerShell/blob/b428e1a393242ce15e54f24c6fcd9d8bae97ada2/src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs#L2548